CDCgov/phdi
# Add new configuration for message parsing to support extended schema

# PULL REQUEST

## Summary
- Added philly_ecr.json

## Related Issue
Fixes #2339 

## Test steps
- Go to `containers/orchestration/app/custom_configs/seed-ecr-viewer-config.json`
- Change line 34 "parsing_schema_name" to philly_ecr.json
- Go to ecr-viewer, run `npm run convert-seed-data`
- See that it succeeds and spits out a data.sql 

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)
